dask/dataframe/tests/test_io.py

```diff
@@ -712,3 +753,10 @@ def test_hdf_globbing():
         tm.assert_frame_equal(res.compute(), pd.concat([df] *  3))
     finally:
         shutil.rmtree(tdir)
+
+
+if __name__ == "__main__":
+    test_read_csv_with_datetime_index_partitions_one()
+    test_read_csv_with_datetime_index_partitions_n()
+    test_from_pandas_with_datetime_index()
+    test_usecols()
```

Comment:
Yeah - I run tests that way, too, but for debugging purposes, with an IDE like pyCharm. That one doesn't break when running py.test. From bash, it's set_trace or nothing, I guess. it's convenient. I will remove if you wish.